contrib/go-redis/redis.v8/redis.go

```diff
@@ -100,16 +100,16 @@ func additionalTagOptions(client redis.UniversalClient) []ddtrace.StartSpanOptio
 
 func (ddh *datadogHook) BeforeProcess(ctx context.Context, cmd redis.Cmder) (context.Context, error) {
 	raw := cmd.String()
-	parts := strings.Split(raw, " ")
-	length := len(parts) - 1
+	length := strings.Count(raw, " ")
 	p := ddh.params
-	opts := []ddtrace.StartSpanOption{
+	opts := make([]ddtrace.StartSpanOption, 0, 5+len(ddh.additionalTags)+1)
```

Comment:
I'm concerned that someone will add an option below later on and forget to change this `5`. It won't be the end of the world, but then it will become confusing where it is coming from, so at least let's add a message here:
```suggestion
	opts := make([]ddtrace.StartSpanOption, 0, 5+len(ddh.additionalTags)+1) // 5 options below
```